source/JABHandler.py

```diff
@@ -1,6 +1,7 @@
+# -*- coding: UTF-8 -*-
 #javaAccessBridgeHandler.py
 #A part of NonVisual Desktop Access (NVDA)
-#Copyright (C) 2006-2007 NVDA Contributors <http://www.nvda-project.org/>
+#Copyright (C) 2007-2017 NV Access Limited, Peter Vágner, Renaud Paquay, Babbage B.V.

```

Comment:
I updated this myself, as it was missing some previous contributors and it's a bit tricky to get the Unicode stuff right for Peter's name. Hope you don't mind.